Fantasy-Quest/Assets/Scripts/Dialogue/SymonSpeaker.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections.Generic;
+using Spine.Unity;
+using UnityEngine;
+
+namespace Dialogue
+{
+    [RequireComponent(typeof(Collider2D))]
+    [RequireComponent(typeof(AudioSource))]
+    [AddComponentMenu("Scripts/Dialogue/Dialogue.SymonSpeaker")]
+    public class SymonSpeaker : DialogueSpeaker
+    {
+        [SerializeField]
+        private SkeletonAnimation skeletonAnimation;
+
+        private void Awake()
```

Comment:
Awake is the same as in DialogueSpeaker. Make in DialogueSpeaker 'protected virtual void Awake()'. In this class you may use base.Awake() and add null check on SkeletonAnimation